torchgeo/datasets/skippd.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SKy Images and Photovoltaic Power Dataset (SKIPP'D)."""
+
+import os
+from typing import Any, Callable, Optional, Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_url, extract_archive
+
+
+class SKIPPD(NonGeoDataset):
+    """SKy Images and Photovoltaic Power Dataset (SKIPP'D).
+
+    The `SKIPP'D dataset <https://purl.stanford.edu/dj417rh1007>`_
+    containes ground-based fish-eye photos of the sky for solar
+    forecasting tasks.
+
+    Dataset Format:
+
+    * .hdf5 file containing images and labels
+    * .npy files with corresponding datetime timestamps
+
+    Dataset Features:
+
+    * fish-eye RGB images (64x64px)
+    * power output measurements from 30-kW rooftop PV array
+    * 1-min interval across 3 years (2017-2019)
+    * 349,372 images under the split key `trainval`
+    * 14,003 images under the split key `test`
```

Comment:
Single backticks will give italics instead of monospace. If this is what you want, you should prob use asterisks instead. Stupid rST...